tests/installation/agama.pm

```diff
@@ -119,13 +119,19 @@ sub run {
     assert_and_click('agama-show-tabs');
 
     # default is just a minimal server style install
-    if (check_var('DESKTOP', 'gnome')) {
+    if (get_var('DESKTOP')) {
```

Comment:
IMO, DESKTOP is a common setting, we don't need add this `if` logic here, but nerver mind, it will not cause any problem.